### PR TITLE
don't use deprecated influxdb methods

### PR DESCRIPTION
**Pull request by tomnis**
`createDatabase`, `describeDatabases` have been deprecated, now we must construct queries instead.

### CODE REVIEW

File: src/main/scala/com/workday/warp/persistence/influxdb/InfluxDBClient.scala [hunk 1]
```diff
@@ -157,9 +157,20 @@ trait InfluxDBClient extends StackTraceFilter with CorePersistenceAware {
...
    */
   def databaseExists(databaseName: String): Try[Boolean] = {
-    InfluxDBClient.client match {
+    val showQuery: Query = new Query("SHOW DATABASES", databaseName)
+    InfluxDBClient.maybeClient match {
       case None => Failure(new WarpConfigurationException(InfluxDBClient.error))
-      case Some(client) => Try(client.describeDatabases.asScala.exists(_.equals(databaseName)))
+      case Some(client) => Try {
+        val results: Seq[QueryResult.Result] = client.query(showQuery).getResults.asScala
+        val databaseNames: Seq[String] = for {
```
Comment (tomnis) [inline src/main/scala/com/workday/warp/persistence/influxdb/InfluxDBClient.scala]: query results come back in nested `ArrayList`

File: src/main/scala/com/workday/warp/persistence/influxdb/InfluxDBClient.scala [hunk 2]
```diff
@@ -169,10 +180,22 @@ trait InfluxDBClient extends StackTraceFilter with CorePersistenceAware {
...
+    val dropQuery: Query = new Query(s"""DROP DATABASE "$database"""", database)
+
+    InfluxDBClient.maybeClient match {
+      case None => Failure(new WarpConfigurationException(InfluxDBClient.error))
+      case Some(client) => Try(client.query(dropQuery))
+    }
+  }
+
+
+  def createDatabase(database: String): Try[Unit] = {
```
Comment (tomnis) [inline src/main/scala/com/workday/warp/persistence/influxdb/InfluxDBClient.scala]: let me add some doc here

**Comment by tomnis** [comment]
i'll cut a 2.0 after this gets merged

File: src/main/scala/com/workday/warp/persistence/influxdb/InfluxDBClient.scala [hunk 3]
```diff
@@ -157,9 +157,20 @@ trait InfluxDBClient extends StackTraceFilter with CorePersistenceAware {
...
-    InfluxDBClient.client match {
+    val showQuery: Query = new Query("SHOW DATABASES", databaseName)
+    InfluxDBClient.maybeClient match {
       case None => Failure(new WarpConfigurationException(InfluxDBClient.error))
-      case Some(client) => Try(client.describeDatabases.asScala.exists(_.equals(databaseName)))
+      case Some(client) => Try {
+        val results: Seq[QueryResult.Result] = client.query(showQuery).getResults.asScala
+        val databaseNames: Seq[String] = for {
+          res <- results
+          serie <- res.getSeries.asScala
```
Comment (tsoppet) [inline src/main/scala/com/workday/warp/persistence/influxdb/InfluxDBClient.scala]: did you want this to be `series` or did you mean to have it as `serie`

Comment (tomnis) [reply src/main/scala/com/workday/warp/persistence/influxdb/InfluxDBClient.scala]: hah its kinda dumb, but intentional. i wanted to clarify that its a single series, a `serie`. `getSeries` returns a `List<Series>`, where a `Series` has these fields:
```java
    private Map<String, String> tags;
    private List<String> columns;
    private List<List<Object>> values;
```

**Pull request merged**